src/pages/episode/sections/show-notes/person-notes.js

```diff
@@ -23,15 +23,13 @@ function PersonNotes({person}) {
       {
         nothing ?
           <p>No links, tips, or picks this week</p> :
-        (
```

Comment:
That's great, but I like this better:

```javascript
{nothing ? (
  <p>No links, tips, or picks this week</p>
) : (
  <div className={css(styles.notesContainer)}>
    {notesSections.map((section, index) => (
      <Notes {...section} key={index} />
    ))}
  </div>
)}
```

My style has slowly changed and solidified to this :)